src/v/security/audit/audit_log_manager.h

```diff
@@ -78,6 +78,19 @@ class audit_log_manager
         return do_enqueue_audit_event(std::make_unique<T>(std::forward<T>(t)));
     }
 
+    template<InheritsFromOCSFBase T>
+    bool enqueue_authn_event(T&& t) {
+        if (!_audit_enabled()) {
+            return true;
```

Comment:
Shouldn't this be `false`?

I don't know what exactly it means to return true or false from this method but it feels like auditing disabled and system shutdown should both not enqueue authn events.

Reply:
Oh I see this return result is not `did_enqueue` but `should log`, which is why it's awkward. I don't really have any better suggestions other than adding a comment about why this is true.